index.bs

```diff
@@ -701,11 +784,20 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 
 				1. If |r| is not "granted", then reject |p| with a "NotAllowedError" DOMException
 
-				1. Let |cleanItemList| be an empty sequence&lt;{{Blob}}>.
+				1. Let |itemList| and |cleanItemList| be an empty sequence&lt;{{Blob}}>.
+
+				1. For each |item| in |data|:
```

Comment:
`data` is of type `ClipboardItems` (plural). So `item` is of type `ClipboardItem`. It seems the data-algorithm defined here would need to iterate over the types.